templates/_layouts/examples.html

```diff
@@ -27,11 +27,16 @@
     {% endif %}
 
     <script defer src="{{ versioned_static('js/example-tools.js') }}"></script>
+    {% else %}
+      <meta name="combined_template_name" content="{{ self._TemplateReference__context.name }}">
```

Comment:
What's the purpose of this? Are we gonna use this info from meta in any way?

Reply:
It was sort of an artifact of testing to make sure that `TemplateReference` was working properly, but no intention of really using it at this point. I've removed it